plugins/amazonq/chat/jetbrains-community/tst/software/aws/toolkits/jetbrains/services/amazonqFeatureDev/FeatureDevTestBase.kt

```diff
@@ -9,6 +9,7 @@ import com.intellij.openapi.project.modules
 import com.intellij.openapi.vfs.VirtualFile
 import com.intellij.testFramework.DisposableRule
 import com.intellij.testFramework.replaceService
+import org.gradle.internal.impldep.com.amazonaws.ResponseMetadata
```

Comment:
dont reference gradle shadow libraries

Reply:
Could you please suggest, how to populate `responseMetadata` then; as I see similar reference in [CodeWhisperer](https://github.com/aws/aws-toolkit-jetbrains/blame/main/plugins/toolkit/jetbrains-core/tst/software/aws/toolkits/jetbrains/services/codewhisperer/CodeWhispererTestUtil.kt#L8) and [Gumby](https://github.com/aws/aws-toolkit-jetbrains/blob/main/plugins/amazonq/codetransform/jetbrains-community/tst/software/aws/toolkits/jetbrains/services/codemodernizer/CodeWhispererCodeModernizerTestBase.kt#L18) tests ?

Reply:
`software.amazon.awssdk.awscore.util.AwsHeader` or declare a const somewhere

they're in codewhisperer and gumby because someone added it and then everyone after copied it